guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,355 @@
+* Tracking scripts should track the cart view correctly
+* Tracking scripts should track an item removed from the cart correctly
+* Tracking scripts should track a discount added to the cart correctly
+* The checkout call-to-action button should be prominent and should be visible without scrolling
+* Cart tap/click targets should be visible, large enough to be tapped/clicked, and unobscured by the edges of the viewport window
+
+### Shopper-Users
+{:.no_toc}
+
+* Shopper-user should be able to log in
+* Shopper-user should be able to sign up
+* Shopper-user should be able to access account
+* Shopper-user should be able to log out
+* Shopper-user should be able to edit information, email, and password
+* When a Shopper-user performs an unsuccessful sign up, log in, or account information submit attempt, they should be presented with error messages that are clear, concise, and accessible to a 5th-grade reading level
+* When a Shopper-user performs an successful sign up, log in, log out, or account information submit attempt, they should be presented with a clear success message
+* A guest shopper-user should be able create an account with a password via an emailed link after checkout
+* A Shopper-user should be able to reset their password via a password reset form
+* When a shopper-user attempts to view another user's order status, they should be shown an error
+
+### Checkout
+{:.no_toc}
+
+* Checkout view should be width-responsive and should behave as expected at phone (landscape and portrait), tablet (landscape and portrait), and desktop resolutions and aspect ratios
+* A shopper should be able to checkout on a single page
+* A shopper should be able to select from multiple methods of payment
+* When a shopper commits an error, helpful error messages should be displayed
+* When a shopper commits an error in a form field an error message should be displayed near the erroneous form field
+* Error messages should be clear, concise, and accessible to a 5th-grade reading level
+* The confirm order button should appear large, prominent, legible, and highly contrasted against its surroundings
+* The confirm order button should be visible to a desktop resolution of 768 pixels high
+* Tracking scripts should correctly track each checkout step performed by the shopper
+* Tap/click targets should be visible, large enough to be tapped/clicked, and unobscured by the edges of the viewport window
+
+### Order Completion
+{:.no_toc}
+
+* Order completion view should be width-responsive and should behave as expected at phone (landscape and portrait), tablet (landscape and portrait), and desktop resolutions and aspect ratios
+* The order number, order line items, and summary should be displayed clearly and in a large font
+* A call to action should be prominently displayed for making order changes
+* Tracking scripts should correctly track completed purchases and views of the order detail
```

Comment:
I'm not certain about this change, so correct me if I'm wrong, but I believe that "detail" in "...views of the order detail" should be plural rather than singular - "...views of the order details". 🤔

Reply:
Good catch. This is all confusing, so I've refactored to this:

> * Tracking scripts should correctly track a completed purchase
> * When the completed purchase is tracked on the order detail view, a subsequent visit to the order detail view should not be tracked as an additionally completed purchase

Reply:
I like that much better. 👍